src/mtxLora/commands/uplink/GetDayEnergies.ts

```diff
@@ -0,0 +1,112 @@
+import Command, {TCommandExampleList, COMMAND_HEADER_SIZE} from '../../Command.js';
+import CommandBinaryBuffer, {IDate, TTariffsEnergies} from '../../CommandBinaryBuffer.js';
+import mergeUint8Arrays from '../../../utils/mergeUint8Arrays.js';
+import {UPLINK} from '../../../constants/directions.js';
+
+
+const COMMAND_ID = 0x78;
+const DATE_SIZE = 3; // year, month, day
+const MAX_TARIFFS_ENERGIES_SIZE = 5 * 4 * 4; // 5 energy types, 4 tariffs, 4 bytes - energy value
+const COMMAND_MAX_SIZE = DATE_SIZE + MAX_TARIFFS_ENERGIES_SIZE;
+
+interface IGetDayEnergies {
```

Comment:
better place interface block before const

in other files as well

Reply:
fixed